src/test/resources/json/addfilemetadata_data_all.json

```diff
@@ -3,7 +3,7 @@
     "addFileMetadata": {
       "filePropertyName": "SHA256ServerSideChecksum",
       "fileId": "07a3a4bd-0281-4a6d-a4c1-8fa3239e1313",
-      "value": "value"
+      "value": "SHA256ClientSideChecksum Value"
```

Comment:
This says `ClientSide` but line 4 is `ServerSide`. Copy-paste error?

Reply:
Ah, or is it this so that it matches the client-side value? I wonder if it would be better to rename it to make it easier to follow.

Reply:
It needs to be `SHA256ServerSideChecksum` as the property name because it's mocking the response getting that value. It is a bit weird having the value set to that though. I've changed it to a random checksum in all the places it's needed. What do you think?